File: frontend/static/script/models/population/Builder.js
"use strict";
import { Worker } from "./Worker.js";



export class Builder extends Worker {
    constructor(population_id, name, profession, alive, employed, field_of_work, working, production) {
        super(population_id, name, profession, alive, employed, field_of_work, working, production);
        
    }
}

File: backend_refactoring/CSharpApi/Models/BluePrints/Beings/Builder.cs
namespace CSharpApi.Models.BluePrints.Beings
{
    public class Builder : Worker
    {
        public Builder(string workerName, bool employed = false, string profession = "worker", bool alive = true, string fieldOfWork = "builder", bool working = false, int production = 500) : base(workerName, employed, profession, alive, fieldOfWork, working, production)
        {
        }
    }
}
